### ISSUE TITLE
train speed suddenly becomes slowly

### ISSUE THREAD

**Issue by kaixinbear** [xingyizhou/CenterNet#630]
I train the kitti dataset with command
`python main.py ddd --exp_id 3dop --dataset kitti --kitti_split 3dop --batch_size 16 --master_batch 7 --num_epochs 70 --lr_step 45,60 --gpus 0,1  `
it goes well in first 10 epoch.but when it started training 11th epoch,
`ddd/3dop |                                | train: [11][0/232]|Tot: 0:00:12 |ETA: 0:00:00 |loss 46.5830 |hm_loss 27.7674 |dep_loss 0.5084 |dim_loss 0.0280 |rot_loss 1.9324 |wh_loss 0.8778 |off_loss 0.0189 |vertex_hm_loss 12.0631 |vertex_coordinate_loss 5.1446 |vertex_off_loss 0.0160 |Data 1.897s(1.897s) |Neddd/3dop |                                | train: [11][1/232]|Tot: 0:00:13 |ETA: 0:48:22 |loss 46.3759 |hm_loss 27.3420 |dep_loss 0.5428 |dim_loss 0.0201 |rot_loss 1.8666 |wh_loss 1.0086 |off_loss 0.0180 |vertex_hm_loss 11.8886 |vertex_coordinate_loss 5.5311 |vertex_off_loss 0.0165 |Data 0.033s(0.965s) |Net 6.604s` 
the screen was frozen stiff with above info.
Can anyone tell me what happened and how to fix it?

**Comment by xingyizhou** [comment]
Thank you for extending it to vertex. However I have never met this. It seems the dataloading takes most of the time. Can you make sure you haven't run other heavy programs on your cpu or check your vertex pre-processing code?

**Comment by kaixinbear** [comment]
I check out programs on the server and find that there is some another process on cpu and gpu(my teammate's).When that process being free up,I can train normally.
